src/course-outline/status-bar/StatusBar.jsx

```diff
@@ -48,109 +70,135 @@ const StatusBar = ({
-        >
-          <Form.Label
-            className="h5"
-          >{intl.formatMessage(messages.videoSharingTitle)}
-          </Form.Label>
+        </StatusBarItem>
+        <StatusBarItem title={intl.formatMessage(messages.pacingTypeTitle)}>
+          <span className="small">
+            {isSelfPaced
+              ? intl.formatMessage(messages.pacingTypeSelfPaced)
+              : intl.formatMessage(messages.pacingTypeInstructorPaced)}
+          </span>
+        </StatusBarItem>
+        <StatusBarItem title={intl.formatMessage(messages.checklistTitle)}>
+          <Hyperlink
+            className="small"
+            destination={checklistDestination()}
+            showLaunchIcon={false}
+          >
+            {checkListTitle} {intl.formatMessage(messages.checklistCompleted)}
+          </Hyperlink>
+        </StatusBarItem>
+        <StatusBarItem title={intl.formatMessage(messages.highlightEmailsTitle)}>
           <div className="d-flex align-items-center">
-            <Form.Control
-              as="select"
-              defaultValue={videoSharingOptions}
-              onChange={(e) => handleVideoSharingOptionChange(e.target.value)}
-            >
-              {Object.values(VIDEO_SHARING_OPTIONS).map((option) => (
-                <option
-                  key={option}
-                  value={option}
-                >
-                  {getVideoSharingOptionText(option, messages, intl)}
-                </option>
-              ))}
-            </Form.Control>
+            {highlightsEnabledForMessaging ? (
+              <span data-testid="highlights-enabled-span" className="small">
+                {intl.formatMessage(messages.highlightEmailsEnabled)}
+              </span>
+            ) : (
+              <Button data-testid="highlights-enable-button" size="sm" onClick={openEnableHighlightsModal}>
+                {intl.formatMessage(messages.highlightEmailsButton)}
+              </Button>
+            )}
             <Hyperlink
-              className="small"
-              destination={socialSharingUrl}
+              className="small ml-2"
+              destination={contentHighlightsUrl}
               target="_blank"
               showLaunchIcon={false}
             >
-              {intl.formatMessage(messages.videoSharingLink)}
+              {intl.formatMessage(messages.highlightEmailsLink)}
             </Hyperlink>
           </div>
-        </Form.Group>
+        </StatusBarItem>
+        {getConfig().ENABLE_TAGGING_TAXONOMY_PAGES === 'true' && (
+          <StatusBarItem title={intl.formatMessage(messages.courseTagsTitle)}>
+            <div className="d-flex align-items-center">
+              <TagCount count={courseTagCount} />
+              { /* eslint-disable-next-line jsx-a11y/anchor-is-valid */ }
+              <a
+                className="small ml-2"
+                href="#"
+                onClick={() => openManageTagsDrawer()}
```

Comment:
Why not make this a button? Additionally, you can simplify the call to `onClick={openManageTagsDrawer}`

Reply:
I was following the Figma design. It is possible to have the same appearance using a button component (that way we can remove the eslint warning).

Fixed the function assignment here: dd7fdc2d7e941ff92815cb5b6678b99aca68b88f

Reply:
Does the figma design specific that it has to be a link? I don't know the advantage of using a link here since it's being used like a button here with an onClick handler and the actual link usage is being supperessed.

Reply:
The figma is here: https://www.figma.com/file/DmUC2e7BAoQsGDq4KgApPa/Content-Tagging-MVP?type=design&node-id=13585-6454&mode=design&t=UXNUlJJKbdiPmD49-4